### ISSUE TITLE
Coordinate Systems are a kind of scales

### ISSUE THREAD

**Issue by lapsedPacifist** [HajoRijgersberg/OM#29]
Should it be possible to define coordinate systems, such as the geographical coordinate systems in OM? These are a kind of scales in two or three dimensions. This might make it possible to convert coordinates between different systems.

**Comment by dr-shorthair** [comment]
Coordinate systems are not scales, as they have a datum as well as a 'scale'. 
The operations that are available on coordinates (offset, relative position) are not the same as the ops available on quantities. 100 degrees Fahrenheit is not twice 50 degrees.

**Comment by lapsedPacifist** [comment]
I agree (although not all coordinate systems would have a datum if you think of celestial coordinate systems) but I think my question should be more like: Should coordinate systems also be supported in OM? This would mean that for geographical coordinate systems we would need to define the datum, and for celestial systems, the epoch, for example.
